gce_image_import_export_tests/test_suites/test_suite_utils.go

```diff
@@ -0,0 +1,98 @@
+//  Copyright 2019 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+// Package testsuiteutils contains e2e tests utils for image import/export cli tools
+package testsuiteutils
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"regexp"
+	"strings"
+	"sync"
+
+	"github.com/GoogleCloudPlatform/compute-image-tools/go/e2e_test_utils/junitxml"
+	"github.com/GoogleCloudPlatform/compute-image-tools/go/e2e_test_utils/test_config"
+)
+
+// TestSuite is image import test suite.
+func TestSuite(
+	ctx context.Context, tswg *sync.WaitGroup, testSuites chan *junitxml.TestSuite,
```

Comment:
minor: args can start in the previous line to save some space. In other places as well